src/plugins/sqlite.js

```diff
@@ -6,7 +6,7 @@ const sqlite3 = require("sqlite3");
 const Helper = require("../helper");
 const Msg = require("../models/msg");
 
-const currentSchemaVersion = 1;
+const currentSchemaVersion = 1520239200;
```

Comment:
Ahah, I should have been more specific, I was picturing a more human-readable way to do timestamp (e.g. `20180307010400` ...à la Rails... I know I know I should stop mentioning it before it bites back 🙂) but at this point in our implementation, it probably doesn't really matter.